english/explore-outside/investigate-marshmallow-structure/say-hello/explore-other-wings/friendly-dragon/continue-wild-ride/continue-wild-ride.md

```diff
@@ -0,0 +1,2 @@
+After an hour you see underneath you a big crowd of green people having an awesome party
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.